xrpl/asyncio/clients/async_websocket_client.py

```diff
@@ -262,7 +262,9 @@ async def send(self: AsyncWebsocketClient, request: Request) -> None:
             raise XRPLWebsocketException("Websocket is not open")
         await self._do_send(request)
 
-    async def _request_impl(self: WebsocketBase, request: Request) -> Response:
+    async def _request_impl(
+        self: WebsocketBase, request: Request, timeout: Optional[float] = None
```

Comment:
```suggestion
        self: WebsocketBase, request: Request, timeout: Optional[float] = None
```
This should be a keyword-only argument, to avoid accidental usage.
Ditto everywhere else.